src/etl/disease_etl.py

```diff
@@ -135,304 +142,320 @@ class DiseaseETL(ETL):
-        
-        query_and_file_list = self.process_query_params(query_list)
+
+        query_and_file_list = self.process_query_params(query_template_list)
         CSVTransactor.save_file_static(generators, query_and_file_list)
         Neo4jTransactor.execute_query_batch(query_and_file_list)
 
     def get_generators(self, disease_data, batch_size, data_provider):
+        '''Creating generators'''
+
         gene_list_to_yield = []
         allele_list_to_yield = []
         evidence_code_list_to_yield = []
         agm_list_to_yield = []
-        withs =[]
+        withs = []
         pge_list_to_yield = []
         xrefs = []
         counter = 0
-        dateProduced = disease_data['metaData']['dateProduced']
+        date_produced = disease_data['metaData']['dateProduced']
 
-        dataProviders = []
+        data_providers = []
 
-        dataProviderObject = disease_data['metaData']['dataProvider']
+        data_provider_object = disease_data['metaData']['dataProvider']
 
-        dataProviderCrossRef = dataProviderObject.get('crossReference')
-        dataProvider = dataProviderCrossRef.get('id')
-        dataProviderPages = dataProviderCrossRef.get('pages')
-        dataProviderCrossRefSet = []
+        data_provider_cross_ref = data_provider_object.get('crossReference')
+        data_provider = data_provider_cross_ref.get('id')
+        data_provider_pages = data_provider_cross_ref.get('pages')
+        data_provider_cross_ref_set = []
 
         #TODO: get SGD to fix their files.
-        if dataProviderPages is not None:
-            for dataProviderPage in dataProviderPages:
-                crossRefCompleteUrl = ETLHelper.get_page_complete_url(dataProvider, ETL.xrefUrlMap,
-                                                                      dataProvider, dataProviderPage)
-
-                dataProviderCrossRefSet.append(ETLHelper.get_xref_dict(dataProvider, dataProvider, dataProviderPage,
-                                                                       dataProviderPage, dataProvider,
-                                                                       crossRefCompleteUrl,
-                                                                       dataProvider + dataProviderPage))
-
-                dataProviders.append(dataProvider)
-                logger.info("data provider: " + dataProvider)
-
+        if data_provider_pages is not None:
+            for data_provider_page in data_provider_pages:
+                cross_ref_complete_url = ETLHelper.get_page_complete_url(data_provider,
+                                                                         ETL.xref_url_map,
+                                                                         data_provider,
+                                                                         data_provider_page)
+
+                data_provider_cross_ref_set.append(ETLHelper.get_xref_dict(\
```

Comment:
can the first item in the list be in line with the function call, I find this more readable than the \.